.eslintrc.cjs

```diff
@@ -221,7 +221,7 @@ module.exports = {
             paths: [
               {
                 name: '@sanity/ui',
-                importNames: ['Tooltip', 'TooltipProps'],
+                importNames: ['Tooltip', 'TooltipProps', 'Tab'],
```

Comment:
We should also include "TabProps" here, so the imports of TabProps are now from `StudioUI`